Sources/Sham/Models/StubRequest.swift

```diff
@@ -23,10 +23,12 @@ public struct StubRequest: Hashable, CustomStringConvertible {
     
     /// The string representation of the request.
     public var description: String {
-        if let method = self.method, let url = self.url {
-            return "\(method): \(url.absoluteString)"
-        } else if let url = self.url {
-            return "ALL: \(url.absoluteString)"
+        if let urlString = self.url?.sortedAbsoluteString {
```

Comment:
To avoid nesting, I'd probably rather we had this as a `guard` at the top. Fortunately the logic here is fairly simple so I'm fine with it if you disagree and think this is clearer.

Reply:
thats how I had it originally, and tests failed because the methods were lowercased... I wouldve guessed the same as you, given the raw values are `case post = "POST"`.... not sure why tehy are lowercased?